### ISSUE TITLE
Conjunctival fornix foreshortening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9975]
**Preferred term label:**
Conjunctival fornix foreshortening

**Synonyms**
Fornix depth loss; shrinkage; obliteration

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0455781
https://www.ncbi.nlm.nih.gov/medgen/C0455780

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mucous membrane pemphigoid

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001091
- label: Conjunctival fornix foreshortening
- parents: [Abnormal conjunctiva morphology (HP:0000502)](https://hpo.jax.org/app/browse/term/HP:0000502)
- pmid: PMID:24314841
- def: Abnormal reduction in depth of the fornix of the conjunctiva. The fornix is the area where the conjunctiva folds back upon itself, thereby creating a recessed space.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9975
- synonyms:
